### PR TITLE
Fixing of (Replace usage of LocalizedIcon in 'multiselect.tsx' with i/span tags #25109)

### PR DESCRIPTION
**Pull request by G26karthik**
#### Summary
This pull request replaces usage of LocalizedIcon in 'multiselect.tsx' with i/span tags #25109.

#### Ticket Link
FIxes  Replace usage of LocalizedIcon in 'multiselect.tsx' with i/span tags #25109 

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
@G26karthik can you please sign the CLA

**Comment by G26karthik** [comment]
check-cla

**Comment by G26karthik** [comment]
> @G26karthik can you please sign the CLA

Sir I have signed CLA and Also checked my name in contributors list.

**Comment by G26karthik** [comment]
Sorry sir I Unknowingly deleted the head repository. I have contacted GitHub support and raised a ticket. so, I hope this will get resolved soon from my side.

**Comment by G26karthik** [comment]
ss

**Comment by G26karthik** [comment]
ss

File: webapp/channels/src/components/multiselect/multiselect.tsx [hunk 1]
```diff
@@ -332,7 +332,7 @@ export default class MultiSelect<T extends Value> extends React.PureComponent<Pr
             noteTextContainer = (
                 <div className='multi-select__note'>
                     <div className='note__icon'>
-                        <LocalizedIcon
+                        <<i/>
                             className='fa fa-info'
                             title={{id: 'generic_icons.info', defaultMessage: 'Info Icon'}}
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/multiselect/multiselect.tsx:337]: we also need to add react-intl here, can you please take a look https://github.com/mattermost/mattermost/issues/25109#issue-1959027425

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/multiselect/multiselect.tsx (hunk 1 above)
Comment (G26karthik) [reply webapp/channels/src/components/multiselect/multiselect.tsx:336]: I dont Exactly know this " add react-intl here"

**Pull request closed without merging**